### PR TITLE
291 climate

### LINKED ISSUE #291: config code climete
- config test coverage
- config code maintainability
- put it in the readme

### PR DESCRIPTION
**Pull request by IuryNogueira**
fix #291 

Adding codeclimate maintainability to readme and configuration.

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: "2"
+plugins:
+  duplication:
+    enabled: true
+  eslint:
+    enabled: true
+  nodesecurity:
+    enabled: true
+exclude_patterns:
+- backend/
```
Comment (edimossilva) [inline .codeclimate.yml:10]: add empty line

**Review by edimossilva — COMMENTED** [review]
can you set the maintainability hook as required?

**Comment by IuryNogueira** [comment]
Hook is configured
Screen Shot 2021-07-09 at 19 04 27

**Comment by IuryNogueira** [comment]
Done! Codeclimate is required! 😃

**Pull request merged**